### PR TITLE
Added Logout Route in Admin API

### LINKED ISSUE #1: Add logout route in admin API
Logout route is missing in admin API routes. I wanted to implement this route.
Please assign me this task under GSSOC 2023.

### PR DESCRIPTION
**Pull request by abhijeetnishal**
Closes #1 

Implemented the logout functionality and logout route in Admin API.

### CODE REVIEW

File: controllers/adminController.js [hunk 1]
```diff
@@ -66,6 +66,10 @@ const controller = {
       res.status(400).json(error.message);
     }
   },
+  //logout functionality
+  logout: async (req, res) =>{
+    res.clearCookie('token').status(200).json('User Logged-out');
+  },
```
Comment (varun-singhh) [inline controllers/adminController.js:72]: response message in json should be small, and no special chars. `user logged out`

**Review by varun-singhh — CHANGES_REQUESTED** [review]

**Comment by abhijeetnishal** [comment]
Hey @varun-singhh I fixed the json typo.

**Review by kaustubhai — APPROVED** [review]

**Comment by kaustubhai** [comment]
@abhijeetnishal you can also implement this route on frontend if you want, lmk if you will be doing it

**Comment by abhijeetnishal** [comment]
@kaustubhai Ok I will implement this on frontend also.

**Review by varun-singhh — APPROVED** [review]

**Review by varun-singhh — APPROVED** [review]

**Pull request merged**